README.md

```diff
@@ -1 +1,183 @@
-# awsome-books
\ No newline at end of file
+# Awesome Book 
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 CC GLOBAL SUMMIT 2023 WEBSITE <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**CC GLOBAL SUMMIT PROJECT** This project that we build is based on a website allowing users to add/remove books from a list in real time.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>Javascript & DOM</li>
+    
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Added a user side feature to add a baaok with the name and author of it.**
+- **Added a user side feature to add a baaok with the name and author of it.**
+- **Added JavaScript to manipulate DOM elements and events.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo(still in development) <a name="live-demo"></a>
+
+
+- [live project](#)
+
+## 🚀 Live Video(still in development) <a name="live-video"></a>
+
+- [live video ](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
```

Comment:
- [ ] Please remove this placeholder text from the Readme, they are there in the template only to help us guide through filling the info and should not be present in the final output